### ISSUE TITLE
Possible to Put multiple items?

### ISSUE THREAD

**Issue by mrchief** [mooyoul/dynamodb-actions#1]
We have a use case where we want to upload a JSON array. That would require parsing the JSON and multiple putitems or a batch write. Is it possible to support that use case?

**Comment by mooyoul** [comment]
Oh, That sounds great. But unfortunately, Github Actions does not support array inputs even YAML can represent array types. Since Input of Github Actions rely on Environment Variable, it won't be supported.

Anyway, It would be nice if we support batchPut (batchWrite) operation. but as i earlier mentioned,  it will require serialized json array input.

Below is candidate interface:

```typescript
type BatchPutItemInput = {
  operation: "batch-put";
  region: string;
  table: string;
  items: string; // JSON-serialized array
}
```

```yaml
# ...
jobs:
  job:
    runs-on: ubuntu-latest
    timeout-minutes: 5
    steps:
      - name: Put DynamoDB Item
        uses: mooyoul/dynamodb-actions@v1.1.3
        env:
          AWS_DEFAULT_REGION: us-east-1
          AWS_REGION: us-east-1
          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
        with:
          operation: batch-put
          region: us-east-1
          table: my-awesome-config
          item: |
            [{ 
              key: "one",
              value: "wow",
              awesome: true,
              stars: 12345
            }, {
              key: "two",
              value: "such a item",
              awesome: true,
              stars: 12345
            }]
```

What do you think about it? Can you share your use cases if possible?

**Comment by mrchief** [comment]
We'd need a few tweaks and they are inter-related:

 - first, an ability to take multiple files or folder as input. 
 - then use AWS marshaller to convert JSON into dynamo db input format https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB/Converter.html#marshall-property

So something like this:

```
 with:
          operation: batch-put
          region: us-east-1
          table: my-awesome-config
          files: ['path/to/fileA.json', 'path/to/fileB.json', ...]
```

**Comment by mooyoul** [comment]
Cool! I never thought that someone wants to use file as input 😅 

If so, I'd like to implement two features:

1. Add file support to put (single) operation
2. Add batchPut operation with file support

At this moment, there's a concern about file path input.
As i mentioned in previous comment, Github Actions does not support array type in Action Input. So, Multiple file input should use comma-delimited string instead. What do you think about Glob?

**Comment by mrchief** [comment]
> What do you think about Glob?

Anything that works - I know the pains of GH actions firsthand :)

> Add file support to put (single) operation

I'm trying to find better evidence but I think that if your JSON file has an array, then AWS `marshall` should be converting it to a `{'L': ...}` format. So in that case, you should be able to Put the items in the whole file in a single op. Otherwise, it'd suck to iterate through the file and do putItme for every array entry.

**Comment by mooyoul** [comment]
Hello @mrchief 👋 

I just released **v1.2.1** (Please note that v1.2.0 was broken due to missing action metadata update. Use v1.2.1 (latest) instead.)

I've added a new `batch-put` operation with file input support (including Glob support).
Take a look and please leave feedback. Thanks! :)
